src/components/MultiCombobox/MultiCombobox.mdx

```diff
@@ -179,6 +181,83 @@ const Example5 = () => {
 <Example5 />;
 ```
 
+A MultiCombobox can always show the placeholder.
+NOTE: you likely want to also pass the `hideLabel` prop as well so you don't have both a label and a placeholder showing at the same time
```

Comment:
I was considering adding an error in the render of the function that would check for this for you and alert you via the console.  I just wanted some feedback on this route first before adding these types of things